test/compose/etc_hosts/setup.sh

```diff
@@ -0,0 +1,2 @@
+cp /etc/hosts $TMPDIR/etc_hosts.backup
+echo -e '\n127.0.0.1 foobar' >> /etc/hosts
```

Comment:
I am not sure I like doing this by default as these tests are also run on local developers systems not just CI.
It also fails as rootless obviously, it is somewhat concerning that this does not cause a test failure and errors are just ignored. cc @edsantiago 

I think it would be a bit safer to bind mount a new hosts file over it: `mount --bind $TMPDIR/new_hosts /etc/hosts` and as rootless you can do this with `podman unshare mount ...`. This allows you to write any content in the hosts file and then you could also `cat /etc/hosts` in the container and check that the entries were not added.

The main advantage of a mount would be that we never leak the entry permanently (i.e. it is gone after a reboot) if the tests were interrupted.

Reply:
I agree. Just updated the test to use bind mount.

Ignoring test errors - I can submit another PR to capture any errors running the test scripts and report them as failures.